pkg/blackboxexporter/blackboxexporter.go

```diff
@@ -0,0 +1,243 @@
+package blackboxexporter
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+	"github.com/openshift/route-monitor-operator/api/v1alpha1"
```

Comment:
nit: I would have seperated the imports to logical grouping-

what is local (inside route-monitor-operator)
external base packages "context"
and third party as I see fit (this can be split into more but that is already up to you

Reply:
Uhm I think those imports are ordered by go imports. Will have another look

Reply:
when you add a newline between imports, they are sorted inside that section

Reply:
since you can't get imports wrong too easy in go, I think it's not worth the effort to order them, but I'll give it a try :D